docs/cloud/projects.md

```diff
@@ -0,0 +1,245 @@
+4. Click **"Create Project"**.
+5. Set the Project Name as desired.
+6. Select the Standard tier.
+7. Set the configuration options as desired.
+8. Click **"Proceed to Checkout"**. You will be taken to a checkout page.
+9. Enter payment information and hit **"Subscribe"**.
+
+:::tip Enterprise Tier
+
+Need even more power, scale, customization and support?\
+Upgrade to an [Enterprise Project](#create-an-enterprise-project).
+
+:::
+
+## Create an Enterprise Project
+
+Enterprise tier offers power and scale to meet any Project's needs, 19
+[Datacenter regions](/cloud/glossary/#data-processing), upgraded [support options](#support-options), and much more.
+[Learn More](/cloud/glossary/#projects)
+
+:::tip Ready to to go Enterprise?
+
+[Contact Sales](https://directus.io/contact/)
+
+:::
+
+## Access a Project
+
+<video alt="Access a Project" loop muted controls autoplay>
+  <source src="https://cdn.directus.io/docs/v9/cloud/projects/projects-20220329A/access-a-project-20220329A.mp4" type="video/mp4">
+</video>
+
+At some point, you will want to login and access the actual managed Project. You can go the Project's URL directly.
+There also are two ways to access a Project from within the Cloud Dashboard. To access a Project from the Cloud
+Dashboard, follow the steps below.
+
+1. Open the Team Menu in the Dashboard Header and click the desired Team.
+2. Click **"Projects"** to enter the Projects Page.\
+   From here, there are two ways to access a Project:
+
+#### From the Projects Page
+
+3. Click <span mi icon>launch</span> on the desired Project. Your Project login page will open in a new tab.
+
+#### From Project Monitor Page
+
+3. Click a Project to enter its Project Monitor Page.
+4. Click **"Open Project"**. Your Project login page will open in a new tab.
+
+## Monitor a Project
+
+<video alt="Project Monitor Page" loop muted controls autoplay>
+  <source src="https://cdn.directus.io/docs/v9/cloud/glossary/glossary-20220322A/monitor-a-project-20220322A.mp4" type="video/mp4">
+</video>
+
+Graphs on the Project Monitor Page display daily Project information over the last month. The four graphs on this page
+may help inform Node configuration decisions.
+
+- **Combined Node Usage**\
+  Total load placed on all Nodes. If CPU or memory are overloaded, the Node could crash! High usage is a sign it is time
+  to scale up.
+- **Horizontal Scaling**\
+  Number of Standby Nodes activated. If one or more Standy Nodes are frequently activated, it may be time to add more Active
+  Nodes.
+- **API Requests**\
+  Number of API requests. This provides basic insight into traffic cycles, including overall volatility. If there are huge
+  spikes in traffic, it may be wise to configure more Standby Nodes.
+- **API Bandwidth**\
+  The read/write bandwidth. If bandwidth is high, it may be beneficial to vertically-scale with Performance Tier or Enterprise
+  Nodes in order to better handle the larger volume of data.
```

Comment:
@rijkvanzanten You may want to fact-check these two things.

Reply:
LGTM!